README.md

```diff
@@ -120,6 +120,54 @@ octopose generate --ignore Tasks
 ```
 This will remove projects from the manfest.
 
+
+### Generate a manifest for a specific project and version
+```
+octopose generate -p Huddle.ABC -v '{\"Huddle.ABC\" : \"1.99.9.999\"}'

```

Comment:
should probably keep examples to be using semver to promote best practice

Reply:
@domgreen so you don't like the new 2020.05.02.345 format?